tools/cli/commands/cli.js

```diff
@@ -13,6 +13,15 @@ import PATH from 'path-name';
  */
 import { chalkJetpackGreen } from '../helpers/styling';
 
+/**
+ * Show us the status of the cli, such as the currenet linked directory.
+ */
+function cliStatus() {
+	console.log(
+		chalkJetpackGreen( 'Jetpack CLI is currently linked to ' + path.join( __dirname, `../../../` ) )
+	);
+	console.log( 'To change the linked directory of the CLI, run `pnpm cli-setup` ' );
```

Comment:
Where would I execute this command? Within the code base of the other checkout? Should we add some clarity here?